superset/commands/report/execute.py

```diff
@@ -254,6 +255,14 @@ def _get_csv_data(self) -> bytes:
         try:
             logger.info("Getting chart from %s as user %s", url, user.username)
             csv_data = get_chart_csv_data(chart_url=url, auth_cookies=auth_cookies)
+            if csv_data:
+                buf = BytesIO()
+                temp_df = pd.read_csv(StringIO(csv_data.decode("utf-8")))
+                if temp_df.columns[0].strip() == "Unnamed: 0":
```

Comment:
Why `Unnamed: 0`? Can we move it out of the function and make it clear why is that the value we're comparing? Are we certain that no matter what the CVS file is, this is the value it gets with numbered columns?

Reply:
>> Are we certain that no matter what the CVS file is, this is the value it gets with numbered columns?

Yes I am sure about, and not able to find a better way to check that index column is present in csv or not

Reply:
okay got your point.